datahub-web-react/src/App.less

```
@@ -6,3 +6,43 @@
   font-style: normal;
   src: local('Mnarope'), url('./fonts/manrope.woff2') format('woff2'), 
```

Should we keep this? By the way, I think there is a typo here.
```suggestion
  src: local('Manrope'), url('./fonts/manrope.woff2') format('woff2'), 
```

---

Good catch, fixed! And I think we should keep it: there's a lot of styling directly from CSS files, and they can eventually use this font.